modules/vacunas/controller/inscripcion-vacunas.events.ts

```diff
@@ -63,3 +63,22 @@ EventCore.on('vacunas:inscripcion:cancelar-vacunado', async ({ prestacion }, req
     }
 });
 
+// Al validar la prestacion de certificación de paciente con riesgo aumentado de COVID-19, inserta la fecha de certificado en la inscripcion
+EventCore.on('vacunas:inscripcion:certificado', async ({ prestacion }, req: Request) => {
```

Comment:
El req no llega por EventCore, y el Schema de inscripción no tiene AuditPlugin así que habría que sacarlo. 
```suggestion
EventCore.on('vacunas:inscripcion:certificado', async ({ prestacion }) => {
```

Reply:
corregido